tests/Unit/User/Application/CommandHandler/PasskeySignInCommandHandlerTest.php

```
@@ -0,0 +1,398 @@
+
+    private function expectClaimedChallenge(PasskeyChallenge $challenge): void
+    {
+        $this->challengeRepository->expects($this->once())
+            ->method('claimActive')
+            ->with(
+                'challenge-id',
+                PasskeyChallenge::PURPOSE_AUTHENTICATION,
+                self::isInstanceOf(DateTimeImmutable::class)
+            )
+            ->willReturnCallback(static function (
+                string $id,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)